src/AidSelectionStep/AidSelectionStep.tsx

```diff
@@ -57,6 +91,8 @@ export const AidSelectionStep: FunctionComponent<AidSelectionProps> = ({ schema,
           ))}
         </Select>
       </div>
+      <NullsInAidWarning schema={schema} aidColumn={aidColumn} />
+      <Divider />
```

Comment:
I would put the Divider further below so that it's rendered conditionally (as it is in current master):

```jsx
<div className="AidSelectionStep-reserved-space">
  {/* Render next step */}
  {aidColumn && (
    <>
      <Divider />
      {children({ aidColumn })}
    </>
  )}
</div>
```